SECURITY.md

```diff
@@ -1,8 +1,31 @@
 # Security Policy
+Chatwoot is looking forward to working with security researchers across the world to keep Chatwoot and our users safe. If you have found an issue in our systems/application, please reach out to us.
 
 ## Reporting a Vulnerability
 
 We use [huntr.dev](https://huntr.dev/) for security issues that affect our project. If you believe you have found a vulnerability, please disclose it via this [form](https://huntr.dev/bounties/disclose). 
+
 This will enable us to review the vulnerability, fix it promptly, and reward you for your efforts.
 
 If you have any questions about the process, feel free to reach out to hello@chatwoot.com.
+
+
+## Out of scope
+
+Please do not peform testing against Chatwoot production services. Use our staging environment at staging.chatwoot.com.
```

Comment:
let advise to spin up a self hosted instance for now. lets set up a dedicated security testing instance later.